README.md

```diff
@@ -292,7 +292,7 @@ The default connection pool size is right for most applications, and in most cas
 
 However sometimes you may want to change your connection pool size due to high throughput or buffered requests. Ideally, to leave room for traffic fluctuations, a connection pool has about twice the number of connections it takes for maximum saturation. Because a connection can handle a maximum of 100 concurrent requests, between 10 and 50 outstanding requests per connection is optimal. The limit of 100 concurrent streams per gRPC connection is enforced in Google's middleware layer, and you are not able to reconfigure this number.
 
-The following steps help you calculate the optimal number of connections in your channel pool using client-side metrics such as those available from [OpenCensus](https://opencensus.io/guides/grpc/java/#examining-metrics).
+The following steps help you calculate the optimal number of connections in your channel pool using client-side metrics.
 
 From your client-side metrics, gather the following information:
```

Comment:
Same here - you can say "Based on the estimate per-client operation rate or qps ..."

Reply:
Sounds good, updated the wording.